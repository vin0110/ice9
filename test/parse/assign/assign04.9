a := f(b);
c := 1 + d(e) * g;
