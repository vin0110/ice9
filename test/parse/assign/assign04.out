ASSIGN:
  ID:'a'
  CALL:
    ID:'f'
  (arglist)
    ID:'b'
ASSIGN:
  ID:'c'
  BINOP(B_ADD):
    ILIT:1
    BINOP(B_MUL):
      CALL:
        ID:'d'
      (arglist)
        ID:'e'
      ID:'g'
